syne_tune/tuner.py

```diff
@@ -168,7 +168,11 @@ def run(self):
             config_space_exhausted = False
             stop_condition_reached = self._stop_condition()
 
-            while not stop_condition_reached:
+            while (
```

Comment:
Could you name the boolean variable and add a comment explaining the logic? For instance:
```suggestion
            # we stop when either the stop condition is reached or when all trials are done if the wait_trial_completion is activated
            should_stop = stop_condition_reached or self.wait_trial_completion_when_stopping and len(running_trials_ids) > 0
            while should_stop:
```